src/components/Thought.js

```diff
@@ -92,11 +93,14 @@ export const Thought = connect(({ cursor, cursorBeforeEdit, expanded, expandedCo
     canDrag: props => {
       const thoughtMeta = meta(pathToContext(props.thoughtsRankedLive))
       const contextMeta = meta(contextOf(pathToContext(props.thoughtsRankedLive)))
+      const globalSort = localStorage['Settings/Global Sort'] || 'None'
+      const isSortEnabled = (contextMeta.sort && contextMeta.sort.Alphabetical) || globalSort === 'Alphabetical'
+      if (isSortEnabled) store.dispatch({ type: 'error', value: `Cannot moved subthoughts of "${ellipsize(headValue(contextOf(props.cursor)))}" while sort is enabled.` })
```

Comment:
I think it might be better to not `dispatch` an error here actually, because `canDrag` is called for every drag check and the user may still be in the process of dragging the thought where they want it to go. Preventing the `drag` is sufficient.

Side note: Be aware that `props.cursor` can be `null`.